pkg/kustomize/kustomize.go

```diff
@@ -110,6 +113,37 @@ func execAdd(fsys filesys.FileSystem, path string, AllTypesFiles map[fileType][]
 	return nil
 }
 
+func remove(s []string, i int) []string {
+	s[i] = s[len(s)-1]
+	return s[:len(s)-1]
+}
+
+func checkPresence(files []string, fsys filesys.FileSystem) ([]string, error) {
+	// we are in the current directory
+	fileCont, err := fsys.ReadFile("kustomization.yaml")
+	if err != nil {
+		return nil, fmt.Errorf("error reading kustomization.yaml")
```

Comment:
I'm not sure if here we have to return an error. If I'm not mistaken if there is no kustomization.yaml file the kustomize edit command will create the file from scratch.  
Maybe is more correct to swallow the error and return the complete array of files because it means that we simply have to add all the files to it.

Reply:
right! well let's filter the error for file not found, if is found and not readable/writable we should return error

Reply:
We checked and I was wrong. Returning the error is the correct thing to do.